### DISCUSSION TITLE
Refresh token and access token expiry time per client

### DISCUSSION THREAD

**Discussion by sagarshah1983** [ory/hydra#2239] [Ideas]
As I see from documentation, access token and refresh token expiry time can be configured at global level, which allows single value for all clients.
Reference: https://www.ory.sh/hydra/docs/guides/token-expiration/

Are there any plans to allow this setting at per-client level, so that different clients can have different timeouts for the access/refresh token issued to them?

Appreciate suggestions!

**Comment by aeneasr** [comment]
There is an issue for that: https://github.com/ory/hydra/issues/1529 :)

**Reply by sagarshah1983** [reply]
Thank you for pointing me to that open ticket!
